CONTRIBUTING.md

```diff
@@ -16,11 +16,11 @@ set the `GHC_VERSION` environment variable:
 GHC_VERSION=ghc881 nix-build
 ```
 
-To find out which GHC versions your nix setup provides:
+To find out which GHC versions our nix setup provides:
 
 ```
-$ nix eval '(builtins.attrNames ((import <nixpkgs> {}).haskell.compiler))'
-[ "ghc822Binary" "ghc844" "ghc863Binary" "ghc865" "ghc881" "ghcHEAD" "ghcjs" "ghcjs86" "integer-simple" ]
+$ nix eval '(builtins.attrNames (import (builtins.fetchTarball (import ./nix/nixpkgs.nix)) {}).haskell.compiler)'
```

Comment:
I just tried that, it doesn't work. Did you mean to commit the file `./nix/nixpkgs.nix`?

Reply:
Hm, I added this file in https://github.com/sebastiaanvisser/clay/pull/213/files#diff-7b06b62d006ecf7b5a6cd3623976d7a6f1522bb1208ae904f0501347faee6b5d

Did you pull the latest?

Reply:
Ah, my bad. I was on an outdated branch. Yes, this works fine :)